test/partialAttribute/nested-html/property-set-before-CEupgrade.html

```diff
@@ -0,0 +1,107 @@
+
+<!doctype html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
+
+
+    <script src="../../../../web-component-tester/browser.js"></script>
+    <!-- <script src="../../helpers/mock-htmlmerger.js"></script> -->
+
+    <script src="../../../../webcomponentsjs/webcomponents-lite.js"></script>
+
+
+    <link rel="import" href="../../../../polymer/polymer.html">
+    <script src="../../helpers/WCT-Polyfill_bugs_workaround.js"></script>
+    <!-- Will be loaded async <link rel="import" href="../../../starcounter-include.html"> -->
+</head>
+
+<body>
+    <div id="connected-fixture">
+        <starcounter-include></starcounter-include>
+    </div>
+    <div id="disconnected-fixture">
+        <starcounter-include></starcounter-include>
+    </div>
+    <script>
+        function createPartial() {
+            return {
+                'scope1': {
+                    'Html': 'scope1Html',
+                    'doesItWork': 'works!'
+                },
+                'scope2': {
+                    'Html': 'scope2Html;,/?:@&=+$',
+                    'doesItWork': 'works!'
+                },
+                'scope3': {
+                    'doesItWork': 'works!'
+                }
+            };
+        };
+        const MERGED_URL = '/sc/htmlmerger?scope1=scope1Html&scope2=scope2Html%3B%2C%2F%3F%3A%40%26%3D%2B%24&scope3=';
+
+        describe('once element with viewModel property set before upgrade is upgraded', function () {
+            var container = document.querySelector('#container'),
+                disconnectedSCInclude, disconnectedPartial, connectedSCInclude, connectedPartial, importedTemplate;
+            before(function(done) {
+                disconnectedSCInclude = document.createElement('starcounter-include');
+                disconnectedSCInclude.viewModel = disconnectedPartial = createPartial();
+                connectedSCInclude = document.querySelector('#connected-fixture > starcounter-include');
+                connectedSCInclude.viewModel = connectedPartial = createPartial();
+                const link = document.createElement('link');
```

Comment:
such a smart technique to test!

Reply:
which one?

Reply:
I mean importing `starcounter-include` later this way to mimic async behaviour.

Reply:
Well, it's actually "the behavior" ;)